src/domain/friendship_status_calculator.rs

```diff
@@ -1,136 +1,16 @@
-use crate::{
-    domain::{
-        error::CommonError, friendship_event::FriendshipEvent, friendship_status::FriendshipStatus,
-    },
-    entities::friendship_history::FriendshipHistory,
-};
+use crate::domain::{friendship_event::FriendshipEvent, friendship_status::FriendshipStatus};
 
-/// Calculates the new friendship status based on the provided friendship event and the last recorded history.
+// Calculates the new friendship status based on the provided friendship event and the last recorded history
+// This method assumes that the transition is valid and auth
```

Comment:
I'd extend the comment to sth like this for more clarity
```suggestion
/// This method assumes that the transition from the last status to the new is valid and the auth user is allowed to perform it
```